hedera-mirror-rest/transactions.js

```diff
@@ -320,6 +324,55 @@ const createTransferLists = (rows) => {
   };
 };
 
+/**
+ * Gets consensus_timestamps of the transactions that include a transfer from the staking reward account 800
+ * transfers are found, the staking_reward_transfers property will be added to the associated transaction object.
+ *
+ * @param transactions transactions list
+ * @return {[]|{number}[]}
+ */
+const getStakingRewardTimestamps = (transactions) => {
+  return _.isNil(transactions)
+    ? []
+    : transactions
+        .filter(
+          (transaction) =>
+            !_.isNil(transaction.crypto_transfer_list) &&
+            transaction.crypto_transfer_list.some(
+              (cryptoTransfer) => cryptoTransfer.entity_id === StakingRewardTransfer.STAKING_REWARD_ACCOUNT
+            )
+        )
+        .map((transaction) => transaction.consensus_timestamp);
+};
+
+/**
+ * Queries for the staking reward transfer list from the transfer list. If staking reward
+ * transfers are found, the staking_reward_transfers property will be added to the associated transaction object.
+ *
+ * @param transactions transactions list
+ * @return {Promise<Map>} Map<ConsensusTimestamp, StakingRewardTransfer>
+ */
+const getStakingRewardTransfers = async (transactions) => {
+  const rewardsMap = new Map();
+  const stakingRewardTimestamps = getStakingRewardTimestamps(transactions);
+  if (_.isNil(stakingRewardTimestamps) || stakingRewardTimestamps.length === 0) {
+    return rewardsMap;
+  }
+
+  const positions = _.range(1, stakingRewardTimestamps.length + 1).map((position) => `$${position}`);
+  const query = `
+    select ${StakingRewardTransfer.CONSENSUS_TIMESTAMP},
+           json_agg(json_build_object(
+             'account', ${StakingRewardTransfer.ACCOUNT_ID},
+             '${StakingRewardTransfer.AMOUNT}', ${StakingRewardTransfer.AMOUNT}))
```

Comment:
nit
```suggestion
           json_agg(json_build_object(
             'account', ${StakingRewardTransfer.ACCOUNT_ID},
             '${StakingRewardTransfer.AMOUNT}', ${StakingRewardTransfer.AMOUNT})) as staking_reward_transfers
```

also change `t.json_agg` to `t.staking_reward_transfers` on line 372

Reply:
Thanks for the input. I have made these changes.